content/zh-cn/blog/_posts/2023-02-03-sig-instrumentation-spotlight.md

```diff
@@ -0,0 +1,298 @@
+---
+layout: blog
+title: "聚光灯下的 SIG Instrumentation"
+slug: sig-instrumentation-spotlight-2023
+date: 2023-02-03
+---
+<!--
+layout: blog
+title: "Spotlight on SIG Instrumentation"
+slug: sig-instrumentation-spotlight-2023
+date: 2023-02-03
+canonicalUrl: https://www.kubernetes.dev/blog/2023/02/03/sig-instrumentation-spotlight-2023/
+-->
+
+<!--
+**Author:** Imran Noor Mohamed (Delivery Hero)
+-->
+**作者**: Imran Noor Mohamed (Delivery Hero)
+
+**译者**: [Kevin Yang](https://github.com/kevin1689-cloud)
+
+<!--
+Observability requires the right data at the right time for the right consumer
+(human or piece of software) to make the right decision. In the context of Kubernetes,
+having best practices for cluster observability across all Kubernetes components is crucial.
+-->
+
```

Comment:
The blank line at line 27 is not necessary. 

The blank line followed by the commented en text is not required. Same below.

Reply:
@windsonsea Done. Please take a look. Thanks!